packages/messaging-api-line/src/LineClient.js

```diff
@@ -389,6 +392,69 @@ export default class LineClient {
       .then(res => res.data, handleError);
 
   /**
+   * Rich Menu
+   *
+   * https://developers.line.me/en/docs/messaging-api/reference/#rich-menu
+   */
+  getRichMenuList = () =>
+    this._axios
+      .get('/richmenu/list')
+      .then(res => res.data.richmenus, handleError);
+
+  getRichMenu = (richMenuId: string) =>
+    this._axios
+      .get(`/richmenu/${richMenuId}`)
+      .then(res => res.data, handleError);
+
+  createRichMenu = (richMenu: RichMenu) =>
+    this._axios.post('/richmenu', richMenu).then(res => res.data, handleError);
+
+  deleteRichMenu = (richMenuId: string) =>
+    this._axios
+      .delete(`/richmenu/${richMenuId}`)
+      .then(res => res.data, handleError);
+
+  getLinkedRichMenu = (userId: string) =>
+    this._axios
+      .get(`/user/${userId}/richmenu`)
+      .then(res => res.data, handleError);
+
+  linkRichMenu = (userId: string, richMenuId: string) =>
+    this._axios
+      .post(`/user/${userId}/richmenu/${richMenuId}`)
+      .then(res => res.data, handleError);
+
+  unlinkRichMenu = (userId: string) =>
+    this._axios
+      .delete(`/user/${userId}/richmenu`)
+      .then(res => res.data, handleError);
+
+  /**
+   * - Images must have one of the following resolutions: 2500x1686, 2500x843.
+   * - You cannot replace an image attached to a rich menu.
+   *   To update your rich menu image, create a new rich menu object and upload another image.
+   */
+  uploadRichMenuImage = (richMenuId: string, image: Buffer) => {
```

Comment:
do we need to support uploading a stream?

Reply:
Maybe later.